flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBase.java

```diff
@@ -330,8 +315,49 @@ public void cancel() {
 	public void open(Configuration configuration) {
```

Comment:
I wonder if it makes sense to move the method above the run() method.
Then its more logical going through the source code.